services/coincap/coincap-base.js

```diff
@@ -0,0 +1,19 @@
+import { BaseJsonService } from '../index.js'
+
+export default class BaseCoincapService extends BaseJsonService {
+  static category = 'other'
+
+  static defaultBadgeData = { label: 'bitcoin' }
+
+  async fetch({ assetId, schema }) {
+    return this._requestJson({
+      schema,
+      url: `https://api.coincap.io/v2/assets/${assetId}`,
```

Comment:
A note here.
There is a rate limit on this API. From https://docs.coincap.io/

> Free Tier (No API Key)
> 200 requests per minute

> Free Tier (API Key)
> 500 requests per minute

I'm not _super_ concerned by this. 200 requests per minute is pretty generous - it would allow this to become quite a popular service (not convinced this is super likely) and there is scope to either request an API key to get a higher limit or tweak the cache lengths to reduce the number of requests we need to make but we should be aware of this.

Reply:
I didn't pay attention to that, I referenced the documentation in the code.